### ISSUE TITLE
BatchDatabaseInitializer construction method changed

### ISSUE THREAD

**Issue by linux-china** [spring-cloud/spring-cloud-dataflow#1213]
In Spring Boot 1.5.x, org.springframework.boot.autoconfigure.batch.BatchDatabaseInitializer  contruct method changed, but TaskConfiguration still use Spring boot 1.4.x API.  Any plan to make data flow compatible with  Spring Boot 1.5.x? 

```
		@Bean
		public BatchDatabaseInitializer batchRepositoryInitializerForDefaultDBForServer(DataSource dataSource,
				Server server) {
			return new BatchDatabaseInitializer();
		}
```

**Comment by sabbyanandan** [comment]
Hi, @linux-china: It's coming. Keep an eye on https://github.com/spring-cloud/spring-cloud-dataflow/pull/1212.

**Comment by sabbyanandan** [comment]
@linux-china: The PR has been merged and it is available in 1.2.0.BUILD-SNAPSHOT - please retry and let us know if you see any issues with it.
